javaagent/build.gradle.kts

```diff
@@ -69,7 +65,20 @@ dependencies {
   baseJavaagentLibs(project(":instrumentation:internal:internal-reflection:javaagent"))
   baseJavaagentLibs(project(":instrumentation:internal:internal-url-class-loader:javaagent"))
 
-  exporterLibs(project(":javaagent-exporters"))
```

Comment:
can we delete this module now?

Reply:
Oops that was the whole point wasn't it. Also ended up defining in javaagent-tooling instead since we already had an exporter there